spec/lib/email/sender_spec.rb

```diff
@@ -247,6 +247,7 @@ def stub_deliver_response(message)
       it "should remove the right headers" do
         email_sender.send
         expect(message.header["X-Discourse-Topic-Id"]).not_to be_present
+        expect(message.header["X-Discourse-Topic-Ids"]).not_to be_present
```

Comment:
Unrelated to this change, this test case was missed out when we added the `X-Discourse-Topic-Ids` header.